src/Clean.sh

```
@@ -1,23 +1,23 @@
 #!/bin/bash
 
-# This batch script is designed for comprehensive cleaning of your project by deleting unnecessary files.
-# It's crucial to close any Integrated Development Environment (IDE), such as vs code, etc., before executing this script to prevent any conflicts or loss of unsaved data.
-# Please note that the commands included in this script are specifically tailored for the Linux/macOS
+# This batch script cleans your project by deleting unnecessary files.
+# It is important to close any IDEs, such as vs for mac, before running this script to prevent conflicts or data loss.
+# The commands in this script are specifically designed for macOS/Linux.
 
-# Runs dotnet clean for each csproj file
+# Runs the dotnet clean command for each .csproj file.
 for csproj in $(find . -name '*.csproj'); do
```

Shellcheck

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)